subprojects/emcee/worker/src/main/kotlin/com/avito/emcee/worker/internal/rest/HttpServer.kt

```diff
@@ -14,12 +14,15 @@ import io.ktor.server.routing.routing
 
 internal class HttpServer(
     private val handlers: List<RequestHandler<*>>,
-    private val debug: Boolean
+    private val debug: Boolean,
+    private val port: Int,
+    private val gracePeriodMs: Long = 500,
```

Comment:
we don't need that

Reply:
I noticed that after failure our worker works for minute more. So I decide to use this properties explicitly
But problem was not in stopping HttpServer